firefox-ios/nimbus.fml.yaml

```diff
@@ -23,7 +23,7 @@ include:
   - nimbus-features/generalFeatures.yaml
   - nimbus-features/homescreenFeature.yaml
   - nimbus-features/messagingFeature.yaml
-  - nimbus-features/microSurveyFeature.yaml
+  - nimbus-features/microsurveyFeature.yaml
```

Comment:
I renamed this file locally but it doesn't seem to change above ^ @adudenamedruby any thoughts?

Reply:
Had to delete and recreate 🤣 (see commits), let me know if there was a better way.